### ISSUE TITLE
example with flow router

### ISSUE THREAD

**Issue by Alino** [alethes/meteor-pages#174]
Please, could someone provide an example how to use basic pagination with flow router?
From the README I cannot understand how to configure it manually by myself when I set options for pages to **router: true** on client side.

**Comment by Alino** [comment]
first bulletpoint in this part of the README is the thing I struggle with here:

> router (String, default = **undefined)** - Three options:
> true - a router is used but the routes are configured separately by the user
> false - no router used
> "iron-router" - iron-router is used and the routes are automatically set up by Pages

**Comment by sungwoncho** [comment]
I'm in the similar situation. Are you using this package successfully with Flow Router?

**Comment by benwalks** [comment]
You can not add the router object and standard pagination should be OK.  I had a hard time getting it to work with infinite scroll though.

**Comment by nicholasalanbrown** [comment]
+1 for a Flow Router (ideally + React.js) example...

**Comment by Alino** [comment]
@sungwoncho unfortunately I am not successful in using this with flow router yet. Example is needed.

**Comment by sungwoncho** [comment]
@Alino Yeah an example would be nice. I ended up implementing my own pagination because my case was simple. I only needed 'prev' and 'next' button not the page numbers.

**Comment by craigbennett1981** [comment]
+1

**Comment by nitya** [comment]
+1

**Comment by veyronfei** [comment]
+1

**Comment by macroramesh6** [comment]
Any work around? Please share.

**Comment by rdewolff** [comment]
Any news?

**Comment by babenzele** [comment]
+1

**Comment by jlourenco** [comment]
+1

**Comment by sibinx7** [comment]
+100

**Comment by wemyss** [comment]
+1

**Comment by thomasmenegotto** [comment]
+1

**Comment by tomyo** [comment]
+1

**Comment by tomyo** [comment]
In the mean while, this minimal package got me going: [https://github.com/miguelalarcos/flow-router-pagination](https://github.com/miguelalarcos/flow-router-pagination). The key is to use [tmeasday:publish-counts](https://atmospherejs.com/tmeasday/publish-counts) package for real time counting the documents (used by the `flow-router-pagination` package).

**Comment by krem06** [comment]
++1

**Comment by vedovato** [comment]
+1

**Comment by wangdengbin** [comment]
+1
